app/models/state_file_md_intake.rb

```diff
@@ -158,6 +158,10 @@ def calculate_age(dob, inclusive_of_jan_1)
     super(dob, inclusive_of_jan_1: false)
   end
 
+  def is_filer_55_and_older?(filer)
```

Comment:
Thought about something like this, but as discussed, letting it fail seems rather than hiding this error seems preferable

```suggestion
  def is_filer_55_and_older?(filer)
     return false unless send(filer)&.birth_date&.present?
     
     calculate_age(send(filer).birth_ate, inclusive_of_jan_1: false) >= 55
```